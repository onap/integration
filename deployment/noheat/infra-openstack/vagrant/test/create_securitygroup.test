#!/bin/sh

export SECURITYGROUP_NAME='onap_ci_lab'

export VAGRANT_CWD='..'

set_up() {
    vagrant up --provision-with=run_playbook_destroy
    vagrant up --provision-with=run_playbook_create
}

check() {
    local secgrp="$1"
    vagrant ssh operator --no-tty -c \
        "export OS_CLOUD=openstack; openstack security group list -fcsv" \
        | grep "$secgrp" \
        | cut -d',' -f2
}

set_up >/dev/null # drop provisioning output
check "$SECURITYGROUP_NAME"
