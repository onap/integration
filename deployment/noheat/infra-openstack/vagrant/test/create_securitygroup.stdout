"onap_ci_lab"
